pages/data/maps/index.vue

```diff
@@ -96,7 +106,25 @@ export default {
           file_path: `${url_list[url_list.length - 1]}`,
         }
       });
+    },
+
+    copyLink() {
+      let url = 'localhost:3000' + this.$router.currentRoute.fullPath;
```

Comment:
localhost:3000 can be replaced by adding a new environment variable BASEURL in nuxt.js.